startup.sh

```diff
@@ -12,10 +12,10 @@
 
 SUBSONIC_HOME=/var/subsonic
 SUBSONIC_HOST=0.0.0.0
-SUBSONIC_PORT=4040
-SUBSONIC_HTTPS_PORT=0
-SUBSONIC_CONTEXT_PATH=/
-SUBSONIC_MAX_MEMORY=200
+SUBSONIC_PORT=${HTTP_PORT}
```

Comment:
Where do these variables come from? Where is "CONTEXT_PATH" defined, for example?

Reply:
What if we just posted a docker-compose along side this as well. I'd be willing to add mine as a PR

To your point about things having to be set, you already have to set things or it wont work (i.e. volumes)

Reply:
I grant the point that we already have to pass something to docker run (for the volume).

Another option would be to keep the ENV statements with sane default. This way, users can override only what they want with the `docker run -e var=value` command.

For example, users won't typically have to override HTTPS_PORT. We could do `ENV HTTPS_PORT=443` inside the file. This would allow most users to just use docker run with the regular parameters without worrying about defining this port. If they need to redefine HTTPS_PORT, all they need to do is run `docker run -e HTTPS_PORT=443 ...` and we're set.

Does it make sense?

Reply:
Yea I think this is fair :+1: :ok_hand: